files/fr/glossary/css_pixel/index.md

```diff
@@ -15,7 +15,7 @@ translation_of: Glossary/CSS_pixel
 original_slug: Glossaire/Pixel_CSS
 ---
 
-Le **pixel CSS**—désigné dans {{Glossary("CSS")}} avec le suffixe `px`—est une unité de longueur qui correspond approximativement à la largeur ou à la hauteur d'un point unique qui peut être vu confortablement par l'œil humain sans effort mais par ailleurs aussi petit que possible. Par définition, il s'agit de la taille physique d'un seul pixel à une densité de pixels de 96 DPI, situé à une longueur de bras des yeux du spectateur.
+Le **pixel CSS**—désigné dans {{Glossary("CSS")}} avec le suffixe `px` — est une unité de longueur qui correspond approximativement à la largeur ou à la hauteur d'un point unique qui peut être vu confortablement par l'œil humain sans effort mais par ailleurs aussi petit que possible. Par définition, il s'agit de la taille physique d'un seul pixel à une densité de pixels de 96 DPI, situé à une longueur de bras des yeux du spectateur.
```

Comment:
```suggestion
Le **pixel CSS** — désigné dans {{Glossary("CSS")}} avec le suffixe `px` — est une unité de longueur qui correspond approximativement à la largeur ou à la hauteur d'un point unique qui peut être vu confortablement par l'œil humain sans effort mais par ailleurs aussi petit que possible. Par définition, il s'agit de la taille physique d'un seul pixel à une densité de pixels de 96 DPI, situé à une longueur de bras des yeux du spectateur.
```
non-blocking/unrelated, but there is another suggestion below so might as well "correct" this